src/Http/Controllers/VoyagerAuthController.php

```diff
@@ -52,6 +52,6 @@ public function postLogin(Request $request)
      */
     public function redirectTo()
     {
-        return route('voyager.dashboard');
+        return config('voyager.user.redirect');
```

Comment:
Can you give the config call a default value?  Without it, this will blow up if the config isn't there (anyone that upgrades from an older version)